src/analyzer/git-caller.server.ts

```diff
@@ -91,6 +91,19 @@ export class GitCaller {
     return await GitCaller._getRepositoryHead(this.repo)
   }
 
+  async gitShow(commits: string[]) {
+    if (!this.branch) throw Error("branch not set")
+    const args = [
+      "show", 
+      "--no-patch", 
+      '--format="author <|%an|> date <|%at|> message <|%s|> body <|%b|> hash <|%H|>"'
+    ]
+
+    commits.forEach(x => args.push(x))
+    const result = (await runProcess(this.repo, "git", args)) as string
```

Comment:
Could the commits just be added to args using the spread operator?

Reply:
indeed they could